test/cpp/jit/test_load_upgraders.cpp

```diff
@@ -0,0 +1,45 @@
+#include <caffe2/serialize/versions.h>
+#include <gtest/gtest.h>
+#include <torch/csrc/jit/api/module.h>
+#include <torch/csrc/jit/operator_upgraders/upgraders.h>
+#include <torch/csrc/jit/operator_upgraders/version_map.h>
+#include <torch/csrc/jit/serialization/import.h>
+
+#include <test/cpp/jit/test_utils.h>
+
+namespace torch {
+namespace jit {
+
+#if ENABLE_UPGRADERS
+// Basic tests to check if C++ torch::jit::load
+// can load the upgraders fine
+// TODO (tugsuu) add more tests
+TEST(UpgraderLoad, CanPopulateUpgradersGraph) {
+  Module m("m");
+  m.define(R"(
+    def forward(self, x: Tensor):
+      b = 5
+      return torch.div(x, b)
+  )");
+  std::stringstream ms;
+  m.save(ms);
+  auto loaded_m = torch::jit::load(ms);
+  auto version_map = get_operator_version_map();
+  auto upgraders = dump_upgraders_map();
+
+  for (const auto& entry : version_map) {
```

Comment:
This test would pass if `version_map` is empty. Is it intended?

Reply:
Yes. This means there are no upgraders available. But I guess I can make it 1-to-1 check